rinks/reston.html

```diff
@@ -0,0 +1,368 @@
+<!DOCTYPE html>
```

Comment:
These rink pages all look good. Nice use of a consistent layout (template) for each one.